src/cclib/io/filewriter.py

```diff
@@ -20,11 +20,15 @@
 from cclib.parser.utils import PeriodicTable
 
 
+class MissingAttributeError(Exception):
+    """Raised when some necessary attribute is not available."""
+
+
 class Writer(object):
     """Abstract class for writer objects.
 
     Subclasses defined by cclib:
```

Comment:
Removed for sure now : ) https://github.com/cclib/cclib/pull/388/commits/3b5e1457f922383f9bf9090db4c94fef0c3c03b9